test/controllers/income/bbsi/BbsiCloseAccountControllerSpec.scala

```diff
@@ -406,6 +408,42 @@ class BbsiCloseAccountControllerSpec extends PlaySpec
 
       status(result) mustBe BAD_REQUEST
     }
+    "no name provided" in {
```

Comment:
The two new tests are verifying an internal server error response, so I think they should be moved into a test block of their own - at the moment they're sitting inside a block which states "return bad request"